tests/cli-ui/rustup-init_help_flag_stdout.toml

```diff
@@ -0,0 +1,27 @@
+bin.name = "rustup-init"
+args = ["--help"]
+status.code = 0
+stdout = """
+rustup-init [..]
+The installer for rustup
+
+USAGE:
+    rustup-init[EXE] [FLAGS] [OPTIONS]
+
+FLAGS:
+    -v, --verbose                        Enable verbose output
```

Comment:
I'm not sure if this will be stable as clap evolves - we may want to do things like

`--verbose[...] Enable verbose output`  : but we can do that if we need to, not right now ;)